### ISSUE TITLE
Detection of triplets that do not exist

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#30]
In a couple of imports I see a tendency of Audiveris to detect triplets where there are non.
I cannot see a systematic reason.
Maybe you have a look at the appended PDF: if I import it, I get a couple of triplets starting from measure 25 and then esp. measure 29ff.
[Londonderry Air.pdf](https://github.com/Audiveris/audiveris/files/1461136/Londonderry.Air.pdf)

[Londonderry_Air.zip](https://github.com/Audiveris/audiveris/files/1461139/Londonderry_Air.zip)

**Comment by hbitteur** [comment]
I just processed your pdf file, and there was no tuplet at all. There are some mistakes, that you cannot immediately fix, because you don't have yet the UI that is coming with 5.1 release.
My guess is that MuseScore introduces these triplets because I found none in Audiveris output.

Have you checked the .omr file itself (it's just a zip file, simply expand it and browse through the .xml files there).
